DigitalLearningSolutions.Data/Services/UserService.cs

```diff
@@ -384,6 +384,12 @@ CentreAnswersData centreAnswersData
                 editDelegateDetailsData.Email
             );
 
+            userDataService.UpdateDelegateProfessionalRegistrationNumber(

```

Comment:
Any unit test to check this is called?

Also, should this update the number for any other linked delegate accounts?

Reply:
Unsure on the other delegates question @SteveJacksonSoft?

Reply:
Let's not update other records right now. After the user account refactor PRN will be specific to a user rather than a delegate account. Let's stick with only updating the name and email for other accounts.